Add blog post about facilitating reviews (#394)
openSUSE/obs-landing: _posts/development/2023-01-25-request-workflow-redesign.md

---
layout: post
title: Request Page Redesign - Facilitating the Review Process
category: development
---

> Collaboration is the heart of the OBS project, therefore, we have been working on the request page redesign for a while,

Reviewer: ```suggestion
Collaboration is the heart of the OBS project. Therefore, we have been working on the request page redesign for a while,
```
Without a dot, it sounds to me like a verly long phrase...

the page where most of the collaboration happens.
This time, we have focused on
improving the handling of requests with multiple actions,
facilitating the review process by enhancing the code changes and
helping out with decision-making, among others.

The request redesign is part of [the beta program](/2018/10/04/the-beta-program/).

{% include partials/_series-of-posts-about-request-redesign.md %}

### Handling Multiple Actions

Sometimes you have to deal with many actions in one single request.

> With the page redesign, you have to move along a dropdown list of actions to select the one you want to review next. It's easy to get lost, we know, that's why we now offer a way to mark those you have already seen.

Reviewer: ```suggestion
With the page redesign, you have to move along a dropdown list of actions to select the one you want to review next. It's easy to get lost, we know. That's why we now offer a way to mark those you have already seen.
```
Also here...

<figure>
  <img src="/images/posts/sprint_133_mark_as_seen.png" alt="A screenshot of the Action Seen button" />
  <figcaption>Mark Action as Seen</figcaption>
</figure>

Apart from that, detecting which is the current action is easier from now on:
  - it is better highlighted in the dropdown list,
  - the dropdown always scrolls towards the current action once you open it,
  - you will always see the position of the action in comparison with the total number of actions

> But a picture is worth a thousand words.

Reviewer: ```suggestion
But a small animation is worth a thousand words:
```
What follows is not really a picture. Feel free to change my suggestion.

Author: What about "_But an image is worth a thousand words:_" ?

Reviewer: Fine with me. :+1:
